com.inform7.IDE.yml

```diff
@@ -0,0 +1,94 @@
+---
+app-id: com.inform7.IDE
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: gnome-inform7
+rename-desktop-file: gnome-inform7.desktop
+build-options:
+  arch:
+    x86_64:
+      env:
+        INFORM7_ARCH: x86_64
+    i386:
+      env:
+        INFORM7_ARCH: i386
+    aarch64:
+      env:
+        INFORM7_ARCH: armv6lhf
+    arm:
+      env:
+        INFORM7_ARCH: armv6lhf
+modules:
+- shared-modules/gtk2/gtk2.json
+- name: webkitgtk
+  config-opts:
+  - --with-gtk=2.0
+  - --disable-webkit2
+  - --disable-geolocation
+  - --disable-video
+  - --disable-spellcheck
+  - --disable-credential-storage
+  - --disable-web-audio
+  build-options:
+    env:
+      PYTHON: /usr/bin/python3
+    arch:
+      aarch64:
+        cflags: -DENABLE_YARR_JIT=0 -fno-delete-null-pointer-checks
+        cxxflags: -DENABLE_YARR_JIT=0 -fno-delete-null-pointer-checks
+        config-opts:
+        - --disable-jit
+  cleanup:
+  - /share/doc
+  - /bin
+  sources:
+  - type: archive
+    url: https://www.webkitgtk.org/releases/webkitgtk-2.4.11.tar.xz
+    sha256: 588aea051bfbacced27fdfe0335a957dca839ebe36aa548df39c7bbafdb65bf7
+  - type: patch
+    path: patches/webkitgtk-2.4.11.patch
+- name: goocanvas
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: http://ftp.gnome.org/pub/GNOME/sources/goocanvas/1.0/goocanvas-1.0.0.tar.gz
+    sha256: 8bd4187f2045080c1f8b6bac135d4231c401e89cb71a30d9c32fd63f4c6245aa
+- name: gtksourceview2
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: https://download.gnome.org/sources/gtksourceview/2.2/gtksourceview-2.2.2.tar.gz
+    sha256: 9330de25198b1d22ca005f8571a8eb22a7074cdd039e63fcbe5b6900ec2acd64
+- name: gtkspell
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: https://sourceforge.net/projects/gtkspell/files/2.0.16/gtkspell-2.0.16.tar.gz
+    sha256: 8fc7dc560167b2cb7193e76aca625a152dc19b0ebf49816b78539cbb90d80d02
+- name: gnome-inform7
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: http://inform7.com/download/content/6M62/I7_6M62_Linux_all.tar.gz
+    sha256: 684e33d37e6fd21a1822233ddf35937f3a365c4a366486a113c5f32015d93cbd
+  - type: archive
+    url: https://github.com/ptomato/gnome-inform7/releases/download/6M62/Gnome_UI_Source_6M62.tar.xz
+    sha256: 6290a0caa0c7c82c44a1cbd338962dbe0f4206d937eec48eafbd6af7366b7ff8
+  - type: patch
+    path: patches/gnome-inform7.patch
+  - type: file
+    path: files/com.inform7.IDE.appdata.xml
+  - type: shell
+    commands:
+    - tar -f inform7-compilers_6M62_${INFORM7_ARCH}.tar.gz -C src/ni/ -x share/inform7/Compilers/ni --strip-components=3
+    - for dir in pixmaps/*x*; do mv ${dir}/apps/{inform7,com.inform7.IDE}.png; done
+    - mv com.inform7.{GUI,IDE}.gschema.xml
+    - mkdir /app/share/metainfo && install com.inform7.IDE.appdata.xml /app/share/metainfo/
+finish-args:
```

Comment:
Those should be put at the top of the manifest instead.

Reply:
Done.